libfwupdplugin/meson.build

```diff
@@ -125,12 +124,19 @@ library_deps = [
   valgrind,
   libjcat,
   platform_deps,
+  efivar,
```

Comment:
this needs to be surrounded by `host_machine.system() == 'freebsd'` no?

Reply:
Done.

Reply:
@hughsie, I've seen there is `platform_deps`. Should it be used in this case or it's not an appropriate usage?

Reply:
I guess you could use that instead. I don't mind either way.